overlay/components/PageHeader.js

```diff
@@ -21,10 +21,12 @@ function PageHeader(document, root, props) {
   pageHeaderContainer.style.boxShadow = '0 1px 4px rgba(0, 0, 0, 0.3)';
   pageHeaderContainer.style.color = '#' + theme.white;
   pageHeaderContainer.style.left = '0';
+  pageHeaderContainer.style.right = '0';
   pageHeaderContainer.style.padding = '1rem 1.5rem';
+  pageHeaderContainer.style.paddingLeft = 'max(1.5rem, env(safe-area-inset-left))';
+  pageHeaderContainer.style.paddingRight = 'max(1.5rem, env(safe-area-inset-right))';
```

Comment:
Graceful fallback is provided in L25 - see https://github.com/pmmmwh/react-refresh-webpack-plugin/pull/528/files#r748701391

Reply:
Would we need this as well? Thinking about it the answer seems to be no since we have a global wrapper outside of the page header?

```js
pageHeaderContainer.style.paddingBottom = 'max(1rem, env(safe-area-inset-bottom))';
pageHeaderContainer.style.paddingTop = 'max(1rem, env(safe-area-inset-top))';
```

Reply:
Not sure. I had no intention of changing the behavior in any way other than expanding paddings when needed if they are smaller than env-area-inset-*.

Reply:
I've tested on a device with a notch and I just realised that the top bar actually never hit the unsafe area due to the browser address bar, so I think we won't need the top overrides 👍🏻

Reply:
Not if you display the webpage as PWA with fullscreen mode :)